.devcontainer/devcontainer.json

```diff
@@ -5,16 +5,8 @@
     "build": {
         "dockerfile": "Dockerfile",
         "context": "..",
+        // ARGS set in ./Dockerfile can be overridden here:
         "args": {
-            // The following values override those set in ./Dockerfile:
-
-            // Update 'VARIANT' to pick a Python version: 3, 3.10, 3.9, 3.8, 3.7, 3.6
-            // Append -bullseye or -buster to pin to an OS version.
-            // Note: If on arm64/Apple Silicon, must use -bullseye variant.
-            "VARIANT": "3.10-bullseye",
-
-            // Options
-            "NODE_VERSION": "lts/*"
         }
```

Comment:
@leifwalsh, thinking more about https://github.com/jab/bidict/pull/237#discussion_r818237905, perhaps their templates are just designed more for e.g. Flask+React web apps that need node? I doubt bidict will ever need node. And I think the templates are just being confusing in this case (e.g. they should not have shipped with a different value of NODE_VERSION in the two files without any explanation for why / what they're' trying to demonstrate there.) SO I think it's better to DRY this up here, and explicitly explain the "overrides Dockerfile" thing (in case that is ever needed in the future), and just accept the divergence from the templates, which I don't think is really that bad in this case. 

As a bonus, leaving "NODE_VERSION" set to "none" reduces the image size by ~150 MB.